src/GCDHelper.cs

```diff
@@ -275,9 +346,9 @@ public unsafe class GCDHelper {
         private bool checkClip;
         private bool checkABC;
         public bool clippedOnThisGCD;
-        private bool clippedOnLastGCD;
-        private bool abcOnThisGCD;
-        private bool abcOnLastGCD;
+        public bool clippedOnLastGCD;
```

Comment:
Uppercase on public variables

Reply:
There's going to be a lot of this -- I wasn't paying attention to what I was doing.  I'll clean it up after vacation.